views/js/qtiCreator/tpl/forms/interactions/choice.tpl

```diff
@@ -63,7 +65,9 @@
     </span>
 </div>
 {{/if}}
+{{#if enabledFeatures.choiceOptionsAvailable}}
 <hr/>
+{{/if}}
```

Comment:
Looks like .tpl files having weird display in GitHub, just in case other reviewers stumble upon it
![image](https://user-images.githubusercontent.com/19402107/166924066-e31c3df5-9921-4ba8-80cb-f1e43e0f2998.png)